src/main/java/com/routeme/api/UserController.java

```diff
@@ -0,0 +1,101 @@
+package com.routeme.api;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.validation.Valid;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.routeme.dto.UserDTO;
+import com.routeme.model.DummyRoute;
+import com.routeme.predictionio.PredictionIOClient;
+import com.routeme.service.UserService;
+
+@RestController
+@RequestMapping(value = "/api/users")
+public final class UserController {
+    private final UserService service;
+    private PredictionIOClient predictionIOClient;
+
+    @Autowired
+    public UserController(UserService service) {
+        this.service = service;
+        this.predictionIOClient = new PredictionIOClient();
+    }
+
+    @RequestMapping(method = RequestMethod.GET, produces = "application/json")
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/heshamhamdymassoud/RouteMe-API/file/2580673864/issues/source?bid=3287637&fileBranchId=3302754#l34)